### ISSUE TITLE
How to visualize significant clusters in searchlight results

### ISSUE THREAD

**Issue by shobrook** [nilearn/nilearn#2558]
Hi,

In the [searchlight example](https://nilearn.github.io/auto_examples/02_decoding/plot_haxby_searchlight.html#sphx-glr-auto-examples-02-decoding-plot-haxby-searchlight-py), there is code for visualizing the p-values produced by a univariate analysis (ANOVA). This is useful, but it would be more useful if we could visualize only the voxels below a certain p-value and above a certain cluster size threshold. Is there a way to do this?

Thanks!

**Comment by bthirion** [comment]
The `cluster_level_inference` function does it.
See in  https://github.com/nilearn/nilearn/blob/master/nilearn/glm/thresholding.py
Does that address your question ?

**Comment by shobrook** [comment]
This function does what I want, except it expects a statistical map of z-values and not p-values.

**Comment by bthirion** [comment]
> Thanks for the response. This function does what I want, except it expects a statistical map of z-values and not p-values.

But it actually works for non z-valued maps. simply notice that only high values are retained.

**Comment by bthirion** [comment]
> Also, how do you import this function? `from nilearn.glm import cluster_level_inference` gives me `AttributeError: module 'nilearn' has no attribute 'glm'`.

You need to clone the Github repository. The code has not been released yet.
Thank you or the reminder: we need to issue a release.

**Comment by NicolasGensollen** [comment]
@shobrook  This should be solved since the `0.7.0` release.
There is also this [example](https://nilearn.github.io/auto_examples/05_glm_second_level/plot_proportion_activated_voxels.html) that is related to the All resolution inference procedure.
@bthirion I was wondering whether we should detail a little bit more this example (or maybe a searchlight example as mentioned above).

**Comment by bthirion** [comment]
Yes, it's always a good idea to make examples more pedgaogical.  
If you have a clear suggestion in mind, please issue a PR. If not, please let me know what you feel is unclear.
